packer/scylla.json

```diff
@@ -156,8 +156,11 @@
         "build_tag": "{{user `build_tag`| clean_resource_name}}",
         "build_mode": "{{user `build_mode`| clean_resource_name}}"
       },
-      "location": "{{user `region`}}",
-      "vm_size": "{{user `vm_size`}}"
+      "vm_size": "{{user `vm_size`}}",
+      "build_resource_group_name": "scylla_images_building_resource_group",
+      "keep_os_disk": true,
+      "virtual_network_name": "scylla_images_building_virtual_network",
+      "private_virtual_network_with_public_ip": true
```

Comment:
I'd go with `scylla_images` for both build_resource_group_name and private_virtual_network_with_public_ip since the object-type stand for itself but it's not critical,
LGTM

Reply:
Good point. changed and verified with https://jenkins.scylladb.com/job/scylla-master/job/releng-testing/job/azure-image/41/

Reply:
```
[yaronkaikov@london]~/git/scylla-machine-image (azure-save-os-disk)$ az image copy --source-resource-group scylla-images --source-object-name debug-scylla-5.3.0-dev-x86_64-2023-02-15T17-02-16 --target-location eastus2 --target-resource-group "scylla-images-eastus2" --target-name debug-scylla-5.3.0-dev-x86_64-2023-02-15T17-02-16
Getting OS disk ID of the source VM/image
Creating source snapshot
Getting sas url for the source snapshot with timeout: 3600 seconds
Target location count: 1
eastus2 - Creating target storage account (can be slow sometimes)
eastus2 - Creating container in the target storage account
eastus2 - Copying blob to target storage account
eastus2 - Copy progress: 0%
eastus2 - Copy progress: 1%
eastus2 - Copy progress: 2%
eastus2 - Copy progress: 3%
eastus2 - Copy progress: 4%
eastus2 - Copy progress: 5%
eastus2 - Copy progress: 6%
eastus2 - Copy progress: 7%
eastus2 - Copy progress: 8%
eastus2 - Copy progress: 9%
eastus2 - Copy progress: 10%
eastus2 - Copy progress: 11%
eastus2 - Copy progress: 12%
eastus2 - Copy progress: 13%
eastus2 - Copy progress: 14%
eastus2 - Copy progress: 15%
eastus2 - Copy progress: 16%
eastus2 - Copy progress: 19%
eastus2 - Copy progress: 28%
eastus2 - Copy progress: 46%
eastus2 - Copy progress: 59%
eastus2 - Copy progress: 70%
eastus2 - Copy progress: 79%
eastus2 - Copy progress: 93%
eastus2 - Copy progress: 100%
eastus2 - Copy time: 0:06:46.343150
eastus2 - Creating snapshot in target region from the copied blob
eastus2 - Creating final image
Image copy finished
```